# Profitable App profiles for App store and Google play markets 

- AIM : To analyze and find mobile app profiles that are profitable for the App Store and Google Play markets
- In this project we consider only for apps that are free to download and install, assuming the main source of revenue are in-app ads. This means that the revenue for any given app is mostly influenced by the number of users that use an app. 
- The goal for this project is to analyze data to help developers understand what kinds of apps are likely to attract more users.

# Opening and Exploring the Data

In [1]:

from csv import reader 
open_googleAppfile = open ('googleplaystore.csv')
open_AppleStoreFile = open('AppleStore.csv')
Google_data = reader(open_googleAppfile)
Apple_data = reader(open_AppleStoreFile)
goo_app_list = list(Google_data)
Appl_list = list(Apple_data)


def explore_data(dataset,start,end,rows_columns = False):
    dataset_slice = dataset[start:end]
    for row in dataset_slice:
        print(row)
        print('\n')
    if (rows_columns):
        print('Number of rows = ' ,len(dataset))
        print('Number of columns = ' ,len(dataset[0]))
explore_data(goo_app_list,0,3,True)
explore_data(Appl_list,0,3,True)

['App', 'Category', 'Rating', 'Reviews', 'Size', 'Installs', 'Type', 'Price', 'Content Rating', 'Genres', 'Last Updated', 'Current Ver', 'Android Ver']


['Photo Editor & Candy Camera & Grid & ScrapBook', 'ART_AND_DESIGN', '4.1', '159', '19M', '10,000+', 'Free', '0', 'Everyone', 'Art & Design', 'January 7, 2018', '1.0.0', '4.0.3 and up']


['Coloring book moana', 'ART_AND_DESIGN', '3.9', '967', '14M', '500,000+', 'Free', '0', 'Everyone', 'Art & Design;Pretend Play', 'January 15, 2018', '2.0.0', '4.0.3 and up']


Number of rows =  10842
Number of columns =  13
['id', 'track_name', 'size_bytes', 'currency', 'price', 'rating_count_tot', 'rating_count_ver', 'user_rating', 'user_rating_ver', 'ver', 'cont_rating', 'prime_genre', 'sup_devices.num', 'ipadSc_urls.num', 'lang.num', 'vpp_lic']


['284882215', 'Facebook', '389879808', 'USD', '0.0', '2974676', '212', '3.5', '3.5', '95.0', '4+', 'Social Networking', '37', '1', '29', '1']


['389801252', 'Instagram', '113954816', 'USD', '0.0', '21615

In [3]:
print(goo_app_list[10473])

['Life Made WI-Fi Touchscreen Photo Frame', '1.9', '19', '3.0M', '1,000+', 'Free', '0', 'Everyone', '', 'February 11, 2018', '1.0.19', '4.0 and up']


- since the category field value is missing we delete this row

In [4]:
del goo_app_list[10473]


In [5]:
explore_data(goo_app_list,1,5,True)

['Photo Editor & Candy Camera & Grid & ScrapBook', 'ART_AND_DESIGN', '4.1', '159', '19M', '10,000+', 'Free', '0', 'Everyone', 'Art & Design', 'January 7, 2018', '1.0.0', '4.0.3 and up']


['Coloring book moana', 'ART_AND_DESIGN', '3.9', '967', '14M', '500,000+', 'Free', '0', 'Everyone', 'Art & Design;Pretend Play', 'January 15, 2018', '2.0.0', '4.0.3 and up']


['U Launcher Lite – FREE Live Cool Themes, Hide Apps', 'ART_AND_DESIGN', '4.7', '87510', '8.7M', '5,000,000+', 'Free', '0', 'Everyone', 'Art & Design', 'August 1, 2018', '1.2.4', '4.0.3 and up']


['Sketch - Draw & Paint', 'ART_AND_DESIGN', '4.5', '215644', '25M', '50,000,000+', 'Free', '0', 'Teen', 'Art & Design', 'June 8, 2018', 'Varies with device', '4.2 and up']


Number of rows =  10841
Number of columns =  13


In [4]:
def identify_duplicate(dataset):
    unique_list = []
    duplicate_list = []
    for app in dataset[1:]:
        name = app[0]
        if name in unique_list:
            duplicate_list.append(name)
        unique_list.append(name)
    return unique_list , duplicate_list

dup_apps = identify_duplicate(goo_app_list)
print(len(dup_apps[1]))
dup_apps = identify_duplicate(Appl_list)
print(len(dup_apps[1]))


    


1181
0


In [5]:
for app in goo_app_list:
    name = app[0]
    if (name == 'Instagram'):
        print(app)
        
for app in Appl_list:
    name = app[1]
    if (name == 'Instagram'):
        print(app)

['Instagram', 'SOCIAL', '4.5', '66577313', 'Varies with device', '1,000,000,000+', 'Free', '0', 'Teen', 'Social', 'July 31, 2018', 'Varies with device', 'Varies with device']
['Instagram', 'SOCIAL', '4.5', '66577446', 'Varies with device', '1,000,000,000+', 'Free', '0', 'Teen', 'Social', 'July 31, 2018', 'Varies with device', 'Varies with device']
['Instagram', 'SOCIAL', '4.5', '66577313', 'Varies with device', '1,000,000,000+', 'Free', '0', 'Teen', 'Social', 'July 31, 2018', 'Varies with device', 'Varies with device']
['Instagram', 'SOCIAL', '4.5', '66509917', 'Varies with device', '1,000,000,000+', 'Free', '0', 'Teen', 'Social', 'July 31, 2018', 'Varies with device', 'Varies with device']
['389801252', 'Instagram', '113954816', 'USD', '0.0', '2161558', '1289', '4.5', '4.0', '10.23', '12+', 'Photo & Video', '37', '0', '29', '1']


In [6]:
dup_apple_apps = identify_duplicate(Appl_list)
print(Appl_list[1])
print(len(dup_apple_apps[1]))

['284882215', 'Facebook', '389879808', 'USD', '0.0', '2974676', '212', '3.5', '3.5', '95.0', '4+', 'Social Networking', '37', '1', '29', '1']
0


In [7]:
reviews_max = {}
for app in goo_app_list[1:]:
    name = app[0];
    n_reviews = float(app[3])
    if (name in reviews_max and reviews_max[name] < n_reviews):
        reviews_max[name] = n_reviews
    if name not in reviews_max:
        reviews_max[name] = n_reviews
        
print(len(reviews_max))        

9659


# Removing Duplicate Entries 

In [8]:
goo_app_list_clean = []
goo_app_list_added = []

for app in goo_app_list[1:]:
    name = app[0]
    n_reviews = float(app[3])
    if (n_reviews == reviews_max[name] and name not in goo_app_list_added):
        goo_app_list_clean.append(app)
        goo_app_list_added.append(name)
        
print(goo_app_list_clean[0:2])
    




[['Photo Editor & Candy Camera & Grid & ScrapBook', 'ART_AND_DESIGN', '4.1', '159', '19M', '10,000+', 'Free', '0', 'Everyone', 'Art & Design', 'January 7, 2018', '1.0.0', '4.0.3 and up'], ['U Launcher Lite – FREE Live Cool Themes, Hide Apps', 'ART_AND_DESIGN', '4.7', '87510', '8.7M', '5,000,000+', 'Free', '0', 'Everyone', 'Art & Design', 'August 1, 2018', '1.2.4', '4.0.3 and up']]


# Removing Non- English Apps

In [9]:
def identify_non_eng_charc(string):
    count = 0;
    for each in string:
        if(ord(each) > 127):
            count = count + 1         
    if count > 3:
        return False     
    else:
        return True


        
#identify_non_eng_charc('爱奇艺PPS -《欢乐颂2》电视剧热播')
#identify_non_eng_charc('Docs To Go™ Free Office Suite')
identify_non_eng_charc('Instachat 😜')


    
   

True

In [10]:
google_eng_apps = []
apple_eng_apps = []

for each in goo_app_list_clean:
    name = each[0]
    if (identify_non_eng_charc(name)):
        google_eng_apps.append(each)
        
for each in Appl_list[1:]:
    name = each[1]
    if (identify_non_eng_charc(name)):
        apple_eng_apps.append(each)
        
explore_data(google_eng_apps,0 ,3,True)
explore_data(apple_eng_apps,0 ,3,True)




        


['Photo Editor & Candy Camera & Grid & ScrapBook', 'ART_AND_DESIGN', '4.1', '159', '19M', '10,000+', 'Free', '0', 'Everyone', 'Art & Design', 'January 7, 2018', '1.0.0', '4.0.3 and up']


['U Launcher Lite – FREE Live Cool Themes, Hide Apps', 'ART_AND_DESIGN', '4.7', '87510', '8.7M', '5,000,000+', 'Free', '0', 'Everyone', 'Art & Design', 'August 1, 2018', '1.2.4', '4.0.3 and up']


['Sketch - Draw & Paint', 'ART_AND_DESIGN', '4.5', '215644', '25M', '50,000,000+', 'Free', '0', 'Teen', 'Art & Design', 'June 8, 2018', 'Varies with device', '4.2 and up']


Number of rows =  9614
Number of columns =  13
['284882215', 'Facebook', '389879808', 'USD', '0.0', '2974676', '212', '3.5', '3.5', '95.0', '4+', 'Social Networking', '37', '1', '29', '1']


['389801252', 'Instagram', '113954816', 'USD', '0.0', '2161558', '1289', '4.5', '4.0', '10.23', '12+', 'Photo & Video', '37', '0', '29', '1']


['529479190', 'Clash of Clans', '116476928', 'USD', '0.0', '2130805', '579', '4.5', '4.5', '9.24.12', '9+'

# Removing paid apps and creating a list of free apps 

In [11]:
android_free_apps =[]
for each in google_eng_apps:
    price = each[7]
    if price == '0' :
        android_free_apps.append(each)

explore_data(android_free_apps,0 ,9,True)



['Photo Editor & Candy Camera & Grid & ScrapBook', 'ART_AND_DESIGN', '4.1', '159', '19M', '10,000+', 'Free', '0', 'Everyone', 'Art & Design', 'January 7, 2018', '1.0.0', '4.0.3 and up']


['U Launcher Lite – FREE Live Cool Themes, Hide Apps', 'ART_AND_DESIGN', '4.7', '87510', '8.7M', '5,000,000+', 'Free', '0', 'Everyone', 'Art & Design', 'August 1, 2018', '1.2.4', '4.0.3 and up']


['Sketch - Draw & Paint', 'ART_AND_DESIGN', '4.5', '215644', '25M', '50,000,000+', 'Free', '0', 'Teen', 'Art & Design', 'June 8, 2018', 'Varies with device', '4.2 and up']


['Pixel Draw - Number Art Coloring Book', 'ART_AND_DESIGN', '4.3', '967', '2.8M', '100,000+', 'Free', '0', 'Everyone', 'Art & Design;Creativity', 'June 20, 2018', '1.1', '4.4 and up']


['Paper flowers instructions', 'ART_AND_DESIGN', '4.4', '167', '5.6M', '50,000+', 'Free', '0', 'Everyone', 'Art & Design', 'March 26, 2017', '1.0', '2.3 and up']


['Smoke Effect Photo Maker - Smoke Editor', 'ART_AND_DESIGN', '3.8', '178', '19M', '50,000+

In [12]:
apple_free_apps =[]

for each in apple_eng_apps:
    price = float(each[4])
    
    if price == 0.0 :
        apple_free_apps.append(each)
   
        
explore_data(apple_free_apps,0 ,3,True) 


['284882215', 'Facebook', '389879808', 'USD', '0.0', '2974676', '212', '3.5', '3.5', '95.0', '4+', 'Social Networking', '37', '1', '29', '1']


['389801252', 'Instagram', '113954816', 'USD', '0.0', '2161558', '1289', '4.5', '4.0', '10.23', '12+', 'Photo & Video', '37', '0', '29', '1']


['529479190', 'Clash of Clans', '116476928', 'USD', '0.0', '2130805', '579', '4.5', '4.5', '9.24.12', '9+', 'Games', '38', '5', '18', '1']


Number of rows =  3222
Number of columns =  16


 # Frequency table to analyze the markets in Play store and app store by genre or category

In [39]:


def freq_table(dataset,index):
    freq_dict = {}
    for each in dataset:
        req_column = each[index]
        if req_column in freq_dict:
            freq_dict[req_column] += 1
        else:
            freq_dict[req_column] = 1
            
    total = 0
    for each in freq_dict:
        total = total + freq_dict[each]
    for each in freq_dict:
        freq_dict [each ] = (freq_dict[each] / total) * 100 
    return freq_dict

def display_table(dataset,index):
    table = freq_table(dataset, index)
    table_display = []
    for key in table:
        key_val_as_tuple = (table[key], key)
        table_display.append(key_val_as_tuple)

    table_sorted = sorted(table_display, reverse = True)
    for entry in table_sorted:
        print(entry[1], ':', entry[0])


result = freq_table(android_free_apps,1)
print(result)



        

{'PHOTOGRAPHY': 2.944494584837545, 'PRODUCTIVITY': 3.892148014440433, 'LIFESTYLE': 3.9034296028880866, 'SHOPPING': 2.2450361010830324, 'COMICS': 0.6204873646209386, 'ENTERTAINMENT': 0.9589350180505415, 'EVENTS': 0.7107400722021661, 'ART_AND_DESIGN': 0.6430505415162455, 'TRAVEL_AND_LOCAL': 2.33528880866426, 'NEWS_AND_MAGAZINES': 2.7978339350180503, 'FINANCE': 3.7003610108303246, 'DATING': 1.861462093862816, 'LIBRARIES_AND_DEMO': 0.9363718411552346, 'BOOKS_AND_REFERENCE': 2.1435018050541514, 'FOOD_AND_DRINK': 1.2409747292418771, 'COMMUNICATION': 3.2378158844765346, 'AUTO_AND_VEHICLES': 0.9250902527075812, 'SOCIAL': 2.6624548736462095, 'HEALTH_AND_FITNESS': 3.0798736462093865, 'VIDEO_PLAYERS': 1.7937725631768955, 'GAME': 9.724729241877256, 'WEATHER': 0.8009927797833934, 'PERSONALIZATION': 3.3167870036101084, 'FAMILY': 18.907942238267147, 'BUSINESS': 4.591606498194946, 'SPORTS': 3.395758122743682, 'TOOLS': 8.461191335740072, 'PARENTING': 0.6543321299638989, 'HOUSE_AND_HOME': 0.823555956678

In [40]:
result = freq_table(apple_free_apps,-5)
print(result)

{'Utilities': 2.5139664804469275, 'Sports': 2.1415270018621975, 'Business': 0.5276225946617008, 'Education': 3.662321539416512, 'Food & Drink': 0.8069522036002483, 'Music': 2.0484171322160147, 'Travel': 1.2414649286157666, 'Photo & Video': 4.9658597144630665, 'Weather': 0.8690254500310366, 'Catalogs': 0.12414649286157665, 'Book': 0.4345127250155183, 'Entertainment': 7.883302296710118, 'Games': 58.16263190564867, 'Medical': 0.186219739292365, 'Finance': 1.1173184357541899, 'Shopping': 2.60707635009311, 'Productivity': 1.7380509000620732, 'Lifestyle': 1.5828677839851024, 'Reference': 0.5586592178770949, 'Social Networking': 3.2898820608317814, 'News': 1.3345747982619491, 'Health & Fitness': 2.0173805090006205, 'Navigation': 0.186219739292365}


# Initial Inferences

- We can see that about 58 % percentage of the free apps(english) belong to Games in **app store**
- wheras entertainment, Photo & Video are followed by Gamaes with a mere 7.8% and 4.69 % respectively 
- We can say that most of the apps are designed for fun but just because there are more apps meant for fun we cant conclude that there are more users for that genre


- We can see that most of the free english apps in **goolgle play store** belong to the genre family constituting of 18 % followed by 'Games' and 'Tools' with 9.7 % adn 8.4 % respectively
- We see a different pattern in Goggle Play store where there is an equla balance of apps for fun and practical uses unlike app store where the fun apps are dominating


# Analyzing App Store by genre

In [25]:
genre_ios = freq_table(apple_free_apps , -5)

for genre in genre_ios:
    total = 0 
    len_genre = 0
    for app in apple_free_apps:
        genre_app = app[-5]
        if genre_app == genre:
            n_ratings = float(app[5])
            total += n_ratings
            len_genre += 1
    avg_user_rating = total / len_genre
    print(genre ,':', avg_user_rating)
    

    

Utilities : 18684.456790123455
Sports : 23008.898550724636
Business : 7491.117647058823
Education : 7003.983050847458
Food & Drink : 33333.92307692308
Music : 57326.530303030304
Travel : 28243.8
Photo & Video : 28441.54375
Weather : 52279.892857142855
Catalogs : 4004.0
Book : 39758.5
Entertainment : 14029.830708661417
Games : 22788.6696905016
Medical : 612.0
Finance : 31467.944444444445
Shopping : 26919.690476190477
Productivity : 21028.410714285714
Lifestyle : 16485.764705882353
Reference : 74942.11111111111
Social Networking : 71548.34905660378
News : 21248.023255813954
Health & Fitness : 23298.015384615384
Navigation : 86090.33333333333


- We can see Navigation apps have highest number of avg reviews , followed by reference and music 
- This can aslo be because there are more number of popular apps in that genre , for eg : google maps might have more number of users and reviewers , music apps such as spotify 
- Reference apps seems to have 74942 ratings 

In [ ]:
for each in apple_free_apps:
    if (each[-5] == 'Reference'):
        print(each[1] ,':' , each[5])
        
        


# Rcommendations 

* From above results it is seen that the Bible and Dictionary.com have the maximum reviews that skewed up the avg ratings of **reference** category
* And also this seems to be a potential app where we can take a popular book and add some interestring features like a dictioanry in it , an audio feature to listen to or additional fun facts about the book one is reading or even a trivia about popoular books might attract users , since there seems to be a lot of users in this genre.
    * The other apps that seem to have potential are
        * Weather apps - users wont stay in the app for long thus our purpose to  put in-app adds is not served here hence its not a great idea 
        * Food & Drink - These are apps that require delivery services to users for which our firm doent have the required budget 
        * Finance - Apps in thsi category require financial knowledge and experts so that teh app abides by the rules and regyulations and also security which at the moment we cant invest or are interested 
        
    

# Analyzing Google Play store by category

In [42]:
category_android = freq_table(android_free_apps,1)
for category in category_android:
    total1 = 0
    len_category1 = 0
    for each in android_free_apps:
        category_app = each[1]
        if category_app == category :
            installs = each[5]
            installs = installs.replace('+','')
            installs = installs.replace(',','')
            total1 += float(installs)
            len_category1 += 1 
    avg_installs = total1/len_category1 
    print(category,':',avg_installs)
        
        
    
    
    
    

PHOTOGRAPHY : 17840110.40229885
PRODUCTIVITY : 16787331.344927534
LIFESTYLE : 1437816.2687861272
SHOPPING : 7036877.311557789
COMICS : 817657.2727272727
ENTERTAINMENT : 11640705.88235294
EVENTS : 253542.22222222222
ART_AND_DESIGN : 1986335.0877192982
TRAVEL_AND_LOCAL : 13984077.710144928
NEWS_AND_MAGAZINES : 9549178.467741935
FINANCE : 1387692.475609756
DATING : 854028.8303030303
LIBRARIES_AND_DEMO : 638503.734939759
BOOKS_AND_REFERENCE : 8767811.894736841
FOOD_AND_DRINK : 1924897.7363636363
COMMUNICATION : 38456119.167247385
AUTO_AND_VEHICLES : 647317.8170731707
SOCIAL : 23253652.127118643
HEALTH_AND_FITNESS : 4188821.9853479853
VIDEO_PLAYERS : 24727872.452830188
GAME : 15588015.603248259
WEATHER : 5074486.197183099
PERSONALIZATION : 5201482.6122448975
FAMILY : 3695641.8198090694
BUSINESS : 1712290.1474201474
SPORTS : 3638640.1428571427
TOOLS : 10801391.298666667
PARENTING : 542603.6206896552
HOUSE_AND_HOME : 1331540.5616438356
EDUCATION : 1833495.145631068
MEDICAL : 120550.6198083067

- Communication category seems to have the highest number of installs equal to 38456119 

In [44]:
for app in android_free_apps:
    if app[1] == 'COMMUNICATION' and (app[5] == '1,000,000,000+'
                                      or app[5] == '500,000,000+'
                                      or app[5] == '100,000,000+'):
        print(app[0], ':', app[5])

WhatsApp Messenger : 1,000,000,000+
imo beta free calls and text : 100,000,000+
Android Messages : 100,000,000+
Google Duo - High Quality Video Calls : 500,000,000+
Messenger – Text and Video Chat for Free : 1,000,000,000+
imo free video calls and chat : 500,000,000+
Skype - free IM & video calls : 1,000,000,000+
Who : 100,000,000+
GO SMS Pro - Messenger, Free Themes, Emoji : 100,000,000+
LINE: Free Calls & Messages : 500,000,000+
Google Chrome: Fast & Secure : 1,000,000,000+
Firefox Browser fast & private : 100,000,000+
UC Browser - Fast Download Private & Secure : 500,000,000+
Gmail : 1,000,000,000+
Hangouts : 1,000,000,000+
Messenger Lite: Free Calls & Messages : 100,000,000+
Kik : 100,000,000+
KakaoTalk: Free Calls & Text : 100,000,000+
Opera Mini - fast web browser : 100,000,000+
Opera Browser: Fast and Secure : 100,000,000+
Telegram : 100,000,000+
Truecaller: Caller ID, SMS spam blocking & Dialer : 100,000,000+
UC Browser Mini -Tiny Fast Private & Secure : 100,000,000+
Viber Mess

- the communication category has an avg number of installs that is skewed up by apps like watsapp ,Facebook ,Messenger, Skype, Google Chrome, Gmail, and Hangouts that have ove ra billion reviews .


In [46]:
under_100_m = []

for app in android_free_apps:
    n_installs = app[5]
    n_installs = n_installs.replace(',', '')
    n_installs = n_installs.replace('+', '')
    if (app[1] == 'COMMUNICATION') and (float(n_installs) < 100000000):
        under_100_m.append(float(n_installs))
        
sum(under_100_m) / len(under_100_m)

3603485.3884615386

- The avg number of installs reduced over 10 times after removing apps with over 100 million installs for the category communication .
- similar pattern is seen for categories such as Video players ( Youtube, Google Play Movies & TV, or MX Player)
- Game genre seems popular but is saturated and we want a different recommendation.

 <br>**The books and reference genre** looks fairly popular as well, with an average number of installs of 8,767,811, this genre has some potential to work well on the App Store.</br>

In [48]:
for app in android_free_apps:
    if app[1] == 'BOOKS_AND_REFERENCE':
        print(app[0], ':', app[5])

E-Book Read - Read Book for free : 50,000+
Download free book with green book : 100,000+
Wikipedia : 10,000,000+
Cool Reader : 10,000,000+
Free Panda Radio Music : 100,000+
Book store : 1,000,000+
FBReader: Favorite Book Reader : 10,000,000+
English Grammar Complete Handbook : 500,000+
Free Books - Spirit Fanfiction and Stories : 1,000,000+
Google Play Books : 1,000,000,000+
AlReader -any text book reader : 5,000,000+
Offline English Dictionary : 100,000+
Offline: English to Tagalog Dictionary : 500,000+
FamilySearch Tree : 1,000,000+
Cloud of Books : 1,000,000+
Recipes of Prophetic Medicine for free : 500,000+
ReadEra – free ebook reader : 1,000,000+
Anonymous caller detection : 10,000+
Ebook Reader : 5,000,000+
Litnet - E-books : 100,000+
Read books online : 5,000,000+
English to Urdu Dictionary : 500,000+
eBoox: book reader fb2 epub zip : 1,000,000+
English Persian Dictionary : 500,000+
Flybook : 500,000+
All Maths Formulas : 1,000,000+
Ancestry : 5,000,000+
HTC Help : 10,000,000+
E

In [50]:
for app in android_free_apps:
    if app[1] == 'BOOKS_AND_REFERENCE' and (app[5] == '1,000,000,000+'
                                            or app[5] == '500,000,000+'
                                            or app[5] == '100,000,000+'):
        print(app[0], ':', app[5])

Google Play Books : 1,000,000,000+
Bible : 100,000,000+
Amazon Kindle : 100,000,000+
Wattpad 📖 Free Books : 100,000,000+
Audiobooks from Audible : 100,000,000+


In [55]:
for app in android_free_apps:
    if app[1] == 'BOOKS_AND_REFERENCE' and (app[5] == '1,000,000+'
                                            or app[5] == '5,000,000+'
                                            or app[5] == '10,000,000+'
                                            or app[5] == '50,000,000+'):
        print(app[0], ':', app[5])

Wikipedia : 10,000,000+
Cool Reader : 10,000,000+
Book store : 1,000,000+
FBReader: Favorite Book Reader : 10,000,000+
Free Books - Spirit Fanfiction and Stories : 1,000,000+
AlReader -any text book reader : 5,000,000+
FamilySearch Tree : 1,000,000+
Cloud of Books : 1,000,000+
ReadEra – free ebook reader : 1,000,000+
Ebook Reader : 5,000,000+
Read books online : 5,000,000+
eBoox: book reader fb2 epub zip : 1,000,000+
All Maths Formulas : 1,000,000+
Ancestry : 5,000,000+
HTC Help : 10,000,000+
Moon+ Reader : 10,000,000+
English-Myanmar Dictionary : 1,000,000+
Golden Dictionary (EN-AR) : 1,000,000+
All Language Translator Free : 1,000,000+
Aldiko Book Reader : 10,000,000+
Dictionary - WordWeb : 5,000,000+
50000 Free eBooks & Free AudioBooks : 5,000,000+
Al-Quran (Free) : 10,000,000+
Al Quran Indonesia : 10,000,000+
Al'Quran Bahasa Indonesia : 10,000,000+
Al Quran Al karim : 1,000,000+
Al Quran : EAlim - Translations & MP3 Offline : 5,000,000+
Koran Read &MP3 30 Juz Offline : 1,000,000+
H

This niche seems to be dominated by software for processing and reading ebooks, as well as various collections of libraries and dictionaries, so it's probably not a good idea to build similar apps since there'll be some significant competition.

<br> We also notice there are quite a few apps built around the book Quran, which suggests that building an app around a popular book can be profitable. It seems that taking a popular book (perhaps a more recent book) and turning it into an app could be profitable for both the Google Play and the App Store markets. </br>

<br> However, it looks like the market is already full of libraries, so we need to add some special features besides the raw version of the book. This might include daily quotes from the book, an audio version of the book, quizzes on the book, a forum where people can discuss the book, etc.</br>

# Conclusions

In this project , data about the App Store and Google Play mobile apps was analyzed with the goal of recommending an app profile that can be profitable for both markets.

Concluded that taking a popular book (perhaps a more recent book) and turning it into an app could be profitable for both the Google Play and the App Store markets. The markets are already full of libraries, so we need to add some special features besides the raw version of the book. This might include daily quotes from the book, an audio version of the book, quizzes on the book, a forum where people can discuss the book, etc